File: BackEnd/LearnWebAPI/Models/Comment.cs
using LearnWebAPI.Models;

namespace BackEnd.Models
{
    public class Comment
    {
        public Guid Id { get; set; }
        public Guid NewsPaperId { get; set; }
        public NewsPaper? NewsPaper { get; set; }
        public Guid UserId { get; set; }
        public User? User { get; set; }
        public string Content { get; set; }
        public DateTime PostTime { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: BackEnd/LearnWebAPI/FrontEnd/src/interface/comment.ts
import { INewsPaper } from '../components/News/PopularNews/model';
import { IUser } from './user';
export interface IComment {
    id: string;
    newsPaperId: string;
    newsPaper: INewsPaper;
    userId: string;
    user: IUser;
    content: string;
    postTime: string;
    isDeleted: false;
}
